scripts/mdf_parser.py

```diff
@@ -72,6 +77,7 @@ def write_dict_to_csv(fname, data_dict, upper_level_key_length=0):
 def main(argv):
     if (len(argv) > 0):
         read_html_data(argv[0] + '/' + argv[1])
+        fix_villages({1105130121: 'ALUE DUA MUKA O', 3506190010: 'SITIMERTO'})
```

Comment:
In this case, @prasastoadi _only found_ issue for those two villages, what about the other data. Did he had already check entire village data?